bpf/bpf_host.c

```diff
@@ -1280,6 +1280,9 @@ int to_netdev(struct __ctx_buff *ctx __maybe_unused)
 		break;
 	}
 
+	send_trace_notify(ctx, TRACE_TO_NETWORK, srcID, 0, 0,
+			  0, ret, 0);
```

Comment:
Sorry, I missed this before. This packet trace will only be executed if the host firewall is enabled. I think we want to move it below, maybe even after the NodePort handling, to ensure we execute it regardless of host firewall setting and only if the packet isn't dropped by policy.

Reply:
Ah makes sense, moved to below. PTAL.